src/integrations/Lemnisk/browser.js

```diff
@@ -73,6 +73,7 @@ class Lemnisk {
     // disabling eslint as message will be there iinn any case
     // eslint-disable-next-line no-unsafe-optional-chaining
     const { traits } = rudderElement.message?.context;
+    traits['isRudderEvents'] = 1;
```

Comment:
is it possible to use boolean value true instead of number 1

Reply:
This is done.